packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -226,7 +256,7 @@ export default function QueueFromGitOps() {
     const version = 'latest';
     const preset = cannonDefInfo?.def.getPreset(ctx);
     setPreviousPackageInput(`${name}:${version}@${preset}`);
-  }, [cannonDefInfo?.def, selectedDeployType]);
+  }, [cannonDefInfo, cannonDefInfo?.def, selectedDeployType]);
```

Comment:
shouldn't `cannonDefInfo` should be converted to a string in some way? to avoid unnecessary re-renders in case the variable reference changes but still has the same value